src/arfsdao.ts

```diff
@@ -98,6 +98,15 @@ export interface ArFSMovePrivateFileResult extends ArFSMoveFileResult {
 	fileKey: FileKey;
 }
 
+export interface ArFSMoveFolderResult {
```

Comment:
It could make sense to make a useful distinction between public and private here in order to reduce the chance of misuse in the future. Consider `ArFSMovePublicFolderResult`.